### PR TITLE
feat: adapt build file for arm64

### PR DESCRIPTION
**Pull request by hyy0322**
/cc @supereagle @zhujian7

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -298,6 +299,32 @@ compile_notary_migrate_patch:
 	@echo "Done."
 
 compile:check_environment compile_adminserver compile_core compile_jobservice compile_registryctl compile_notary_migrate_patch
+
+compile_adminserver_arm64:
+	@echo "compiling binary for adminserver (golang image)..."
+	@echo $(GOBASEPATH)
+	@echo $(GOBUILDPATH)
+	$(DOCKERCMD) run --rm -v $(BUILDPATH):$(GOBUILDPATH) -w $(GOBUILDPATH_ADMINSERVER) $(GOBUILDIMAGEARM64) $(GOIMAGEBUILD) -o $(GOBUILDMAKEPATH_ADMINSERVER)/$(ADMINSERVERBINARYNAME)
```
Comment (supereagle) [inline Makefile]: Use `if-else` to merge the command for different archs. 

For example:
```
    ifeq ($(ARCH), "arm64")
		$(DOCKERCMD) run --rm -v $(BUILDPATH):$(GOBUILDPATH) -w $(GOBUILDPATH_ADMINSERVER) $(GOBUILDIMAGEARM64) $(GOIMAGEBUILD) -o $(GOBUILDMAKEPATH_ADMINSERVER)/$(ADMINSERVERBINARYNAME)
    else
		$(DOCKERCMD) run --rm -v $(BUILDPATH):$(GOBUILDPATH) -w $(GOBUILDPATH_ADMINSERVER) $(GOBUILDIMAGE) $(GOIMAGEBUILD) -o $(GOBUILDMAKEPATH_ADMINSERVER)/$(ADMINSERVERBINARYNAME)
    endif
```

**Comment by supereagle** [comment]
@hyy0322  Any updates?

File: Makefile (hunk 1 above)
Comment (hyy0322) [reply Makefile]: updated

**Comment by hyy0322** [comment]
> @hyy0322 Any updates?

@supereagle  PTAL

**Comment by supereagle** [comment]
LGTM

**Comment by zhujian7** [comment]
@hyy0322  考虑给该 repo 配个 CI ？

**Pull request merged**